dists/soundfonts/COPYRIGHT.VintageDreamsWaves

```diff
@@ -0,0 +1,21 @@
+ Vintage Dreams Waves v 2.0. for Creative Labs' AWE Soundcards
+ (EMU Soundfont 2 Format)
+
+ Copyright (c) Ian Wilson, 1996  (Updated January 1998)
+
+ This soundfont is freeware. You may freely use and/or redistribute it subject
+ to the following terms:
+ 1. It is not altered, edited, modified, ripped, or converted to other formats,
+ except for private use only.
```

Comment:
This looks like it would cause an issue with the DFSG and similar polices.

I notice that the libretro backend currently packages [a GPLv3 licensed soundfont](https://github.com/scummvm/scummvm/tree/master/backends/platform/libretro/dist). Could that be used on all platforms instead?

Reply:
Thanks. I didn't know we already had a soundfont in the code base. It might indeed be better to reuse that one.